features/step_definitions/networking.rb

```diff
@@ -696,3 +685,29 @@
     raise "Fail to delete the ovs pod"
   end
 end
+
+Given /^the default interface on nodes is stored in the#{OPT_SYM} clipboard$/ do |cb_name|
+  ensure_admin_tagged
+  step "I select a random node's host"
```

Comment:
Can we move this step "I select a random node's host" to case level?

Reply:
@huiran0826 what do you mean for case level?  or any shortcoming for current steps?

Reply:
@zhaozhanqi    Yes, it may cause some inconsistent under some conditions. For example: In  egressip, there are several steps need to execute on the node, such as get default interface ,get subnet for default interface, add valid ip on the node.  So here the node should be the same one, if using "I select a random node's host " in step definition, cannot assure the result from same node. So suggest  move this step to case : Given I select a random node's host

Reply:
mostly this step should be only executed once in one case, and after this step, you can store the node  to clipboard.  
you mean `get subnet for default interface, add valid ip on the node.`  need to use same step?

Reply:
I mean, get default interface, get subnet for default interface, get valid ip on the node, basically all of these steps need to run command on the nodes.  Do you mean just use "I select a random node's host" in get default interface, but not in other steps?  If so ,  get default interface has to be the first step.  Actually why we just move this step "I select a random node's host" to the case?  If still not clear, let's discuss it offline.

Reply:
@huiran0826 Ok, I got you.  you want use this step after doing some changes in one node.  do we have this kind of scenario? if so. we can discuss how to deal with it. since if we move it out. we need to update some cases

Reply:
@zhaozhanqi Yes, I have this kind of scenario, we can discuss it offline.

Reply:
What you guys discuss offline and think, let me know :)